index.html

```diff
@@ -1,258 +1,228 @@
-              sign-ups required.
-            </p>
-            <ul class="flex project_coding_langs">
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-            <button class="preview_button outlinings">See Project</button>
-          </article>
-          <article class="card flex white-back">
-            <img
-              src="./images/project_2.png"
-              alt="Multi-Post project preview"
-            />
-            <h3>Multi-Post Stories</h3>
-            <ul class="flex project_details">
-              <li>CANOPY</li>
-              <li>BACKEND DEV</li>
-              <li>2015</li>
-            </ul>
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul class="flex project_coding_langs">
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-            <button class="preview_button outlinings">See Project</button>
-          </article>
-          <article class="card flex white-back">
-            <img src="./images/project_3.png" alt="tonic project preview" />
-            <h3>Tonic</h3>
-            <ul class="flex project_details">
-              <li>CANOPY</li>
-              <li>BACKEND DEV</li>
-              <li>2015</li>
-            </ul>
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul class="flex project_coding_langs">
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-            <button class="preview_button outlinings">See Project</button>
-          </article>
-          <article class="card flex white-back">
-            <img
-              src="./images/project_4.png"
-              alt="Multi-Post project preview"
-            />
-            <h3>Multi-Post Stories</h3>
-            <ul class="flex project_details">
-              <li>CANOPY</li>
-              <li>BACKEND DEV</li>
-              <li>2015</li>
-            </ul>
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul class="flex project_coding_langs">
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-            <button class="preview_button outlinings">See Project</button>
-          </article>
-        </section>
-        <section class="about white-back">
-          <div>
-            <h1>
-              About <br />
-              Myself
+        </nav>
+    </header>
+    <br><br>
+    <div class="hero-holder">
+    <section class="hero">
+        <div class="container">
+            <h1 class="greetings">
+                I’m Umoru, <br/> Glad to see you!
```

Comment:
- [ ] Great job working on this milestone! 💯 However, You have some issues regarding your styles that are mismatching the FIGMA design:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/221076766-c915551e-c66e-409d-9c4b-2a8a4077e800.png) | ![image](https://user-images.githubusercontent.com/77069011/221076706-c4612ddc-78fb-4bd4-b135-76f2a3b66028.png) |

- [ ] Please consider fixing the color of your links in the mobile version (In desktop they look fine) 🤓 🔧